eventorganizer.api/src/main/java/com/event/organizer/api/model/dto/FeedbackRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package com.event.organizer.api.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class FeedbackRequestDto {
+    private final Integer min = 1;
+    private final Integer max = 5;
+    private Integer rating;
+    private String comment;
+    private Long eventId;
+
+    public Integer setBoundedRating(Integer rating) {
```

Comment:
This should be attached to the Setter of "rating". Is it currently attached? 

In C# normally you can say something like
public int Rating { get; set {

if(value >= min && value <= max) 
{
this = value
}
}}

Reply:
Just checked how to do this in Java, you only need to rename "setBoundedRating" to "setRating"